index.js

```diff
@@ -22,14 +21,22 @@ const exec = (cmd, args) => {
 	).filter(Boolean);
 };
 
+const install = (bin, useYarn) => () => execa(bin, ['install'])
```

Comment:
I finally got into implementing these issues :tada:. I even used the `yarn` with `npm` fallback as example of the feature. https://github.com/SamVerschueren/listr#enabling-tasks